python/ray/utils.py

```diff
@@ -390,6 +390,36 @@ def setup_logger(logging_level, logging_format):
     logger.propagate = False
 
 
+def open_log(path, **kwargs):
+    kwargs.setdefault("buffering", 1)
+    kwargs.setdefault("mode", "a")
+    kwargs.setdefault("encoding", "utf-8")
```

Comment:
We don't anywhere else right? If this is actually required can it be in another PR?

Reply:
I just don't want to one-off change this particular log file encoding and not the others.

Reply:
Actually can we just remove this file and call `open(file, "a", buffering=1)` directly? This is same as the other comment to inline thin wrappers.

Reply:
```
When no buffering argument is given, the default buffering policy works as follows:

Binary files are buffered in fixed-size chunks; the size of the buffer is chosen using a heuristic trying to determine the underlying device’s “block size” and falling back on io.DEFAULT_BUFFER_SIZE. On many systems, the buffer will typically be 4096 or 8192 bytes long.

“Interactive” text files (files for which isatty() returns True) use line buffering. Other text files use the policy described above for binary files.
```
https://docs.python.org/3/library/functions.html#open